### PR TITLE
fix: local development

### PR DESCRIPTION
**Pull request by EverlastingBugstopper**
- don't require SENTRY_DSN locally as the library handles its existence
- run tsc --watch concurrently with netlify dev

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,11 +5,12 @@
   "main": "index.js",
   "scripts": {
     "build": "tsc",
+    "clean": "rm -rf node_modules tscsonfig.tsbuildinfo dist",
```
Comment (abernix) [inline package.json:8]: To try and maintain support with Windows outside of WSL, might we use?:

```suggestion
    "clean": "git clean -dfqX -- ./node_modules ./tscsonfig.tsbuildinfo ./dist",
```

We do this elsewhere, but I'm curious if it matters.  Is that worth trying?

File: package.json [hunk 2]
```diff
@@ -5,11 +5,12 @@
...
     "watch": "tsc --watch",
-    "start": "netlify dev",
+    "start": "npm run build && concurrently 'npm run watch' 'netlify dev'",
     "test": "jest"
   },
-  "author": "",
+  "author": "opensource@apollographql.com",
   "license": "ISC",
```
Comment (abernix) [inline package.json:14]: Unrelated: Should this be `MIT`?

**Review by abernix — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (EverlastingBugstopper) [reply package.json:14]: it likely should be! ISC is just the `npm --init` default

**Review by JakeDawkins — APPROVED** [review]
++ to Jesse's comments too, but this looks great!

File: package.json (hunk 1 above)
Comment (EverlastingBugstopper) [reply package.json]: done!

**Comment by EverlastingBugstopper** [comment]
mmmm, so this still doesn't work i think. i _thought_ the library would handle the absence of `SENTRY_DSN` but it turns out it does not

**Comment by EverlastingBugstopper** [comment]
ok, latest commit should _actually_ actually fix this 😄

**Pull request merged**